### PR TITLE
To do  structure

### PR DESCRIPTION
**Pull request by jaamanu**
Add id to remove button 
Add checkbox tick for task
Add codes to main container

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+},
+{
+  description: 'Be smart and consistent',
+  bool: 'false',
+  index: '2',
+}, {
+  description: 'You can do it James ',
+  bool: 'false',
+  index: '3',
+}];
```
Comment (rloterh) [inline src/index.js:19]: - [ ] Your to-do list is empty! In this activity we want the hard coded task array to be rendered dynamically to the DOM during page load. And the tasks should appear in order of the index values for each task. Kindly ensure to improve upon the same.

**Your to-do list:** 
image

**Expected:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi James, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Webpack correctly configured


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. Howe
ver, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

**You can also consider:**
- Kindly remove redundant live demo link in your ReadMe
- kindly remove guide instructions on how to fix MIT License.
Implementing the above change(s) will make your ReadMe appear more professional.

<br>
Cheers and Happy coding!👏 💻 🍷
<br><br>
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (peterrobert) [reply src/index.js:18]: This change has been implemented well.

**Review by peterrobert — APPROVED** [review]
Hi @jaamanu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**